libraries/WebServer/examples/AdvancedWebServer/AdvancedWebServer.ino

```diff
@@ -44,8 +44,9 @@ void handleRoot() {
   digitalWrite(led, 1);
   char temp[400];
   int sec = millis() / 1000;
-  int min = sec / 60;
-  int hr = min / 60;
+  int hr = sec / 3600;
+  int min = (sec / 60) % 60;
+  sec = sec % 60
```

Comment:
missing `;` at the end of the line

Reply:
BTW looking at the code below, both `min` and `sec` are `% 60`, which makes the new code in effect the same as the old one. Also the old one looks OK, so I wonder how come you got the wrong values?